hw/bsp/espressif/boards/espressif_p4_function_ev/board.h

```diff
@@ -8,7 +8,7 @@
  * in the Software without restriction, including without limitation the rights
  * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
  * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
+ * furnished to do so, subject to the following conditions:aaaaa
```

Comment:
!

Reply:
must be caused when testing with hid_composite example